setup.cfg

```diff
@@ -43,16 +43,19 @@ package_dir =
 # Version specifiers like >=2.2,<3.0 avoid problems due to API changes in
 # new major versions. This works if the required packages follow Semantic Versioning.
 # For more information, check out https://semver.org/.
+# Note: pyparsing is actually needed by QKeras, but missing as dependency
+# tf2onnx 1.9.3 first version after fixing https://github.com/onnx/tensorflow-onnx/issues/1707
```

Comment:
perhaps remove this comment or move/change it to reflect the reason for the `tf2onnx>=1.12.1` pin later in the file?